components/NewRealmWizard/components/steps/InviteMembersForm.tsx

```diff
@@ -251,7 +276,7 @@ export default function InviteMembersForm({
               {inviteList.map((address, index) => (
                 <InviteAddress
                   key={address}
-                  address={address}
+                  address={address + index}
```

Comment:
Why do we want this?

Reply:
Ah, i could take this out. I needed it for the invalid list since it could include duplicates, but for valid addresses, you're right, it's not needed